### PR TITLE
V0.1.0/home Pagination

### PR DESCRIPTION
**Pull request by SpQuyt**

### CODE REVIEW

**Comment by namdaoduy** [comment]
Please merge master and resolve conflict

File: src/components/Home/EventCard/index.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ class EventCard extends Component {
           ribbonColor: 'orange',
           ribbonText: 'white',
         };
-      case 'custom admin':
+      case 'custom team':
```
[the remark this answers is not in the document]
Comment (SpQuyt) [reply src/components/Home/EventCard/index.js]: I have created **rolename.js** in folder **constants**

File: src/datalayer/actions/event.action.js [hunk 2]
```diff
@@ -5,9 +5,9 @@ export const changeToLoginComponent = () => ({
   type: authAction.IS_LOGIN_COMPONENT,
 });
 
-export const loadEventsListDispatch = (uid) => ({
+export const loadEventsListDispatch = (uid, page) => ({
   type: eventAction.LOAD_EVENTS_LIST,
-  promise: get(`/users/${uid}/events`, {}, true),
+  promise: get(`/users/${uid}/events?page=${page}`, {}, true),
```
Comment (namdaoduy) [inline src/datalayer/actions/event.action.js]: As API spec, It would be `p`, not `page`

Comment (SpQuyt) [reply src/datalayer/actions/event.action.js]: Ok.

**Pull request merged**